src/Elastic.Apm/Helpers/IntExtensions.cs

```diff
@@ -15,5 +16,11 @@ public static void Repeat(this int repeatCount, Action<int> action)
 			for (var i = 0; i < repeatCount; ++i)
 				action(i);
 		}
+
+		public static async Task Repeat(this int repeatCount, Func<int, Task> action)
```

Comment:
I think every usage from this class is only in test projects. Would it maybe better to move it to `Elastic.Apm.Tests`? 

...if I overlooked some prod usage, then nevermind.

Reply:
Yes, `IntExtensions` is used only in tests - I've moved it to `Elastic.Apm.Tests`.